src/main/kotlin/no/nav/familie/ef/personhendelse/handler/UtflyttingHandler.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.personhendelse.handler
+
+import no.nav.familie.ef.personhendelse.client.OppgaveClient
+import no.nav.familie.ef.personhendelse.client.SakClient
+import no.nav.familie.ef.personhendelse.client.defaultOpprettOppgaveRequest
+import no.nav.familie.kontrakter.ef.felles.StønadType
+import no.nav.person.pdl.leesah.Personhendelse
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class UtflyttingHandler(val sakClient: SakClient,
+                        val oppgaveClient: OppgaveClient) {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    fun handleUtflytting(personhendelse: Personhendelse) {
+
+        personhendelse.utflyttingFraNorge?.let {
+            logger.info("Mottatt utflyttingshendelse")
+        } ?: throw Exception("Ingen utflyttingFraNorge tilordning i personhendelse : ${personhendelse}")
+
+        val personIdent = personhendelse.personidenter.map { it.toString() }.first()
```

Comment:
Her er det mulig å brukte first direkte, har gjort om de til å være string i stedet for CharSequence :) 
```suggestion
        val personIdent = personhendelse.personidenter.first()
```